tests/components/onewire/test_sensor.py

```diff
@@ -0,0 +1,43 @@
+"""Tests for 1-Wire sensor platform."""
+from homeassistant.components.onewire.const import DEFAULT_SYSBUS_MOUNT_DIR
+import homeassistant.components.sensor as sensor
+from homeassistant.setup import async_setup_component
+
+from tests.common import assert_setup_component
+
+
+async def test_setup_minimum(hass):
+    """Test setup with minimum configuration."""
+    config = {"sensor": {"platform": "onewire"}}
+    with assert_setup_component(1, "sensor"):
+        assert await async_setup_component(hass, sensor.DOMAIN, config)
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6333224)